app/src/main/java/com/github/se/cyrcle/ui/report/ReviewReportScreen.kt

```diff
@@ -55,21 +47,22 @@ fun ReviewReportScreen(
   val configuration = LocalConfiguration.current
   val screenWidth = configuration.screenWidthDp.dp
   val screenHeight = configuration.screenHeightDp.dp
-  val showDialog = remember { mutableStateOf(false) }
-
-  // Define padding as a percentage of screen dimensions
-  val horizontalPaddingScaleFactor = screenWidth * 0.03f
-  val topBoxHeight = screenHeight * 0.10f // 10% of screen height for top box
-  val verticalPaddingScaleFactor = screenHeight * 0.02f
+  val horizontalPadding = screenWidth * 0.03f
+  val topBoxHeight = screenHeight * 0.10f
```

Comment:
maybe you can put constants here for 0.10f like `TopBarScreenRate`

Reply:
Done! Good idea!